Installation of latest LTS node version with `lts/*` alias of `.nvmrc` not working

**Describe the bug**
When executing a `rtx install` inside a dir (`repo`) where we have a `.nvmrc` with a version alias `lts/*` (which is a valid version alias regarding the [nvm doc](https://github.com/nvm-sh/nvm#long-term-support)) I get the following error:

```sh
rtx node@lts/* [ERROR] /Users/user/Library/Caches/rtx/node/node-build/bin/node-bui ✗ 0s rtx /Users/user/Library/Caches/rtx/node/node-build/bin/node-build exited with non-zero status: exit code 2
rtx Run with RTX_DEBUG=1 for more information
```

When just performing it with `lts`, everything works fine. I set the `legacy_version_file` to `true` in the `.rtx.toml` within my home folder.

**To Reproduce**
Try to execute `rtx install` inside a project with a `.nvmrc` containing `lts/*` version alias.

**Expected behavior**
I installs the latest LTS version.

**`rtx doctor` output**

```
rtx version:
  1.35.7 macos-arm64 (unknown 2023-07-30)

build:
  Target: aarch64-apple-darwin
  Features: BREW, DEFAULT, NATIVE_TLS
  Built: Sun, 30 Jul 2023 23:56:12 +0000
  Rust Version: rustc 1.71.0 (8ede3aae2 2023-07-12) (built from a source tarball)
  Profile: release

shell:
  /opt/homebrew/bin/fish
  fish, version 3.6.1

rtx data directory:
  /Users/user/.local/share/rtx

rtx environment variables:
  RTX_SHELL=fish

settings:
  {"always_keep_download": "false", "always_keep_install": "false", "asdf_compat": "false", "disable_default_shorthands": "false", "disable_tools": "[]", "experimental": "false", "jobs": "4", "legacy_version_file": "true", "legacy_version_file_disable_tools": "[]", "log_level": "INFO", "missing_runtime_behavior": "warn", "plugin_autoupdate_last_check_duration": "10080", "raw": "false", "trusted_config_paths": "[]", "verbose": "false"}

config files:
  /Users/user/.rtx.toml
  /Users/user/repo/.nvmrc

plugins:
  go       (core)
  java     (core)
  node     (core)
  python   (core)
  ruby     https://github.com/rtx-plugins/rtx-ruby.git#ff2640f

toolset:
  node@lts/*

```

--- a/src/plugins/core/node.rs
+++ b/src/plugins/core/node.rs
@@ -210,7 +210,10 @@ impl Plugin for NodePlugin {
     fn parse_legacy_file(&self, path: &Path, _settings: &Settings) -> Result<String> {
         let body = fs::read_to_string(path)?;
         // trim "v" prefix
-        Ok(body.trim().strip_prefix('v').unwrap_or(&body).to_string())
+        let body = body.trim().strip_prefix('v').unwrap_or(&body);
+        // replace lts/* with lts
+        let body = body.replace("lts/*", "lts");
+        Ok(body.to_string())
     }
 
     fn external_commands(&self) -> Result<Vec<Command>> {

